src/Microsoft.IdentityModel.Tokens/Encryption/AuthenticatedEncryptionProvider.cs

```diff
@@ -32,6 +33,7 @@ private struct AuthenticatedKeys
         private DecryptionDelegate DecryptFunction;
         private EncryptionDelegate EncryptFunction;
         private const string _className = "Microsoft.IdentityModel.Tokens.AuthenticatedEncryptionProvider";
+        internal const string _skipValidationOfAuthenticationTagLength = "Switch.Microsoft.IdentityModel.SkipAuthenticationTagLengthValidation"; 
```

Comment:
Super nit: can we add "Name" or "Switch" to the end of this variable? If I look at just the variable name "skipValidationOfAuthenticationTagLength" I would expect it to be a bool that tells me whether to skip this or not rather than the name of a switch.

Reply:
For consistency with our other context switch, I might keep is as is. https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/pull/2421/files